packages/react-components/src/theming/icons.tsx

```diff
@@ -78,5 +82,14 @@ export const DEFAULT_COMPONENT_ICONS = {
   MessageEdit: <Edit20Regular />,
   MessageRemove: <Delete20Regular />,
   HorizontalGalleryLeftButton: <ChevronLeft20Regular />,
-  HorizontalGalleryRightButton: <ChevronRight20Regular />
+  HorizontalGalleryRightButton: <ChevronRight20Regular />,
+  errorBarCallNetworkQualityLow: <WifiWarning20Filled />,
+  errorBarCallNoSpeakerFound: <SpeakerMute16Filled />,
```

Comment:
Noticed there are some 16 size icons and some 20 size icons - I also noticed it also happens above in existing code(only happens for once), is it on purpose or they don't provide us some 20-size icons

Reply:
We want 16px icons for the message bar where the text is also smaller. I actually need a 16px icon for `WifiWarning`, but don't know how to resize (yet).